modules/auxiliary/gather/vmware_vcenter_vmdir_ldap.rb

```diff
@@ -19,7 +19,7 @@ def initialize(info = {})
           6.7U3f update.
         },
         'Author' => [
-          # Discovered by unknown researcher(s)
+          'Hynek Petrak', # Discovered by, added hash dumping
```

Comment:
Hey, do you have a blog post or something you can link to for your discovery/analysis of CVE-2020-3952? Thanks!

Reply:
I just reported my analysis to VMWare and completely forgot about it as I was busy with other topics. In the meantime the other guys blogged, so not sure if it’s worth it to write more. The only think that has not been spotted yet, is the technique to dump the hashes and perhaps Kerberos tokens. And I don’t have a blog 🤦🏻‍♂️

Reply:
Hah, all good. Thank you!